docs/configuration-reference.md

```diff
@@ -2061,6 +2061,8 @@ remote_write:
     automatic_logging:
       # indicates the Loki instance to write logs to.
       loki_name: <string>
+      # send all automatic logging to the default logger. This overrides loki_name.
```

Comment:
Personal preference, feel free to ignore: I try to avoid adding configuration like

```
enable_feature_a: true
enable_feature_b: false
enable_feature_c: true
```

And instead try to do something like

```
behavior: "choice_a"
```

So maybe in this case the setting would be something like:

```
automatic_logging:
  backend: "loki"
```

or

```
automatic_logging:
  backend: "stdout"
```

Reply:
yeah, i agree it's not ideal. i went this route b/c it's non-breaking.

@rfratto thoughts on a breaking change to this config? I'm guessing all of 0 people use it.

Reply:
I think that the Agent being v0 and with a small entry in the migration guide, it's fine to have the breaking change.

Reply:
Yeah, breaking changes for this is fine IMO as long as it's in the migration guide. This is all a bit on the experimental side, so there should be some expectation that we'll need to frequently change things. 

If we're not calling this out as experimental in the docs, can you add that?

Reply:
Added experimental tag and improved config.